command/acl/policy/read/policy_read.go

```diff
@@ -66,20 +67,26 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
-	var policyID string
+	var pol *api.ACLPolicy
 	if c.policyID != "" {
-		policyID, err = acl.GetPolicyIDFromPartial(client, c.policyID)
+		c.policyID, err = acl.GetPolicyIDFromPartial(client, c.policyID)
```

Comment:
```suggestion
		policyID, err := acl.GetPolicyIDFromPartial(client, c.policyID)
```
We should try to avoid mutating `c.policyID` so we can guarantee that the fields in `c` come from our input `args` and are not modified by our own code.

Reply:
ACK. totally makes sense. Updated according.
FYI, I am intentionally avoiding short assignment operator (`:=`) to avoid issues with variable shadowing in Go. The last (policy read) error (`err`) of the "if block" has been handled outside if.
Thanks for the review : )

Reply:
I see your point! In practice I've rarely seen issues with variable shadowing around `err` because it's always handled close to assignment so I'd prefer the conciseness of `:=` here but it's a very minor opinion and I'll go ahead and approve your PR!